pensieve/json_dump.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os
+
+CONCEPT_MAP = {'people': 'Person',
+               'places': 'Place',
+               'things': 'Thing',
+               'activities': 'Activity',
+               'times': 'Time'}
+
+
+def dump_mem_to_json(mem_dict, save=None):
+    """
+    Convert mem_dict into a JSON file following the schema and write
+
+    Args:
+        mem_dict: dictionary of mem information
+        save: path to save JSON to [default: None]
+
+    Returns:
+        mem_json: JSON object for memory
+    """
+    node = {'name': '',
+            'label': '',
+            'imageURL': mem_dict['img_url'],
+            'iconURL': '',
+            'created': '',
```

Comment:
This should have a datetime object...for this set we can use utc time
```
from datetime import datetime
created = updated = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.000Z')
```

'2017-07-30T03:03:03.000Z'